microovn/ovn/commands.go

```diff
@@ -16,12 +16,14 @@ const defaultDBConnectWait = 30 //Default time to wait for connection to ovsdb
 const OvsdbConnected = "connected"
 const OvsdbRemoved = "removed"
 
-// ovsdbSpec is a helper structure for precise identification of ovsdb databases. A lot of
-// ovn/ovs commands take path to either database file, database socket or process control socket
-// along with the database name. This structure can be used for such cases.
-type ovsdbSpec struct {
-	Target string
-	Name   string
+// OvsdbSpec is a helper structure that encapsulates properties of an OVN/OVS database.
+type OvsdbSpec struct {
+	SocketPath   string // Path to a socket that the database has open
```

Comment:
While I see that the code is currently hard coded for Unix sockets, it feels limiting knowing the underlying tooling accepts any type of target.

OTOH, if we will only ever use this for sockets i guess its fine and we can always change it later.

Reply:
I see what you mean and it's a good point. The reason I wanted to change it from the previous name `Target` was that some commands, like `ovn-appctl`, take "target" argument which points to `.ctl` file as opposed to `.sock`. I wanted to avoid confusions there.

I'll think of some better name, and I'm open to suggestions. Perhaps a `LocalSocketPath`?

If we have ever a need for using tcp/ssl "targets", we can extend this struct later.

Reply:
Come to think about this, the `LocalSocketPath` wouldn't be the best option. If we ever need to use network targets for the database connection, we could probably add new `OvsdbType`, something like `OvsdbTypeNBCluster` and the constructor for `OvsdbSpec` would populate this field with correct tcp/ssl URLs. So perhaps this field could be called `SocketURL` and it would work for `unix:`, `tcp:` and `ssl:`